src/query/functions/src/scalars/timestamp/src/datetime.rs

```diff
@@ -700,18 +701,29 @@ fn register_to_string(registry: &mut FunctionRegistry) {
                 if format.is_empty() {
                     output.push_null();
                 } else {
-                    // Can't use `tz.timestamp_nanos(self.as_() * 1000)` directly, is may cause multiply with overflow.
-                    let (mut secs, mut nanos) =
-                        (micros / MICROS_PER_SEC, (micros % MICROS_PER_SEC) * 1_000);
-                    if nanos < 0 {
-                        secs -= 1;
-                        nanos += 1_000_000_000;
-                    }
-                    let ts = ctx.func_ctx.tz.timestamp_opt(secs, nanos as u32).unwrap();
                     // https://github.com/BurntSushi/jiff/issues/155
-                    // ASCII is currently required in jiff crate
-                    let res = ts.format(format).to_string();
-                    output.push(&res);
+                    // Consider use jiff crate 0.1.24 version.
+                    // let datetime = micros.to_timestamp(ctx.func_ctx.jiff_tz.clone());
+                    // match strtime::format(&datetime, format) {
+                    //     Ok(res) => output.push(&res),
+                    //     Err(e) => { ctx.set_error(output.len(), e.into());output.push_null() },
+                    // }
```

Comment:
this comment code seems useless, we can delete them.

Reply:
Yes. better to convert as an issue.